provider/runbook_resource.go

```diff
@@ -191,11 +199,17 @@ func createResourceFireHydrantRunbook(ctx context.Context, d *schema.ResourceDat
 	for _, currentStep := range steps {
 		step := currentStep.(map[string]interface{})
 
+		if step["repeats"].(bool) == true && step["repeats_duration"].(string) == "" {
+			return diag.Errorf("Error creating runbook, step repeats requires repeat_duration to be set")
+		}
```

Comment:
same check addition here as in update
```suggestion
		if step["repeats"].(bool) == true && step["repeats_duration"].(string) == "" {
			return diag.Errorf("step repeats requires step repeats_duration to be set")
		}
		if step["repeats"].(bool) == false && step["repeats_duration"].(string) != "" {
			return diag.Errorf("step repeats_duration requires step repeats to be set to true")
		}
```